style.css

```diff
@@ -1 +1,217 @@
-/*  Add your styling here */
+/* General body */
+
+body{
+    background-color: rgb(255, 255, 255);
+    font-family: Arial !important;
+}
+
+/* Header / Upper part */
+
+header{
+    width:100%;
+    display: flex;
+    justify-content: space-between;
+    background-color: rgb(255, 212, 229);
+    color: rgb(134, 134, 134);
+    margin-bottom: 20px;
+}
+
+.bakery-icon{
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+}
+
+.bakery-icon p{
+    font-size: calc(15px + 1vw);
+}
+
+.cupcake-icon img{
+    width:125px;
+    margin-top: 10px;
+
+}
+
+.welcome-msg{
+    flex-wrap: nowrap;
+    margin-top: 90px;
+    width: 100%;
+    float: left;
+    justify-content: right;
+    text-align: end;
+    font-weight: bold;
+}
+
+/* Navigation bar */
+
+#links{
+    display:none;
+}
+
+.hamburger-icon{
+    margin-top:16px;
+    left: -10px;
+    width: 80%;
+    
+}
+.menu-nav {
+    overflow: hidden;
+    background-color:rgb(207, 144, 155);
+    position: absolute;
+    display:none;
+    min-width: 100vw;
+    top:calc(65px + 3vw);
```

Comment:
The `top` property only works in conjunction with `position: absolute` or `position: fixed`